src/view/com/composer/text-input/TextInput.tsx

```diff
@@ -92,6 +96,8 @@ export const TextInput = forwardRef(function TextInputImpl(
        * @see https://github.com/bluesky-social/social-app/issues/929
        */
       setTimeout(async () => {
+        const wasPaste = newText.length > prevLength.current + 1
```

Comment:
This is more like `mayBePaste` right? You could have multiple added characters for other reasons, like maybe accepting an autocomplete suggestion or an autofix.

Reply:
That's a good point. Let me verify that it doesn't actually matter (it shouldn't, since your cursor won't be next to the URL if you're doing an auto correct) 👍

Reply:
Oky, we are not relying on it being a paste, but going to update the wording to make it more clear here.